ts/features/bonus/siciliaVola/screens/voucherGeneration/SickCheckIncomeScreen.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from "react";
+import { useRef } from "react";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import { SafeAreaView, Text } from "react-native";
+import { NavigationEvents } from "react-navigation";
+import { isSome } from "fp-ts/lib/Option";
+import BaseScreenComponent from "../../../../../components/screens/BaseScreenComponent";
+import { emptyContextualHelp } from "../../../../../utils/emptyContextualHelp";
+import { setAccessibilityFocus } from "../../../../../utils/accessibility";
+import { IOStyles } from "../../../../../components/core/variables/IOStyles";
+import { H1 } from "../../../../../components/core/typography/H1";
+import { GlobalState } from "../../../../../store/reducers/types";
+import {
+  svGenerateVoucherBack,
+  svGenerateVoucherCancel,
+  svGenerateVoucherFailure,
+  svGenerateVoucherSelectCategory,
+  svGenerateVoucherUnderThresholdIncome
+} from "../../store/actions/voucherGeneration";
+import FooterWithButtons from "../../../../../components/ui/FooterWithButtons";
+import { SvBeneficiaryCategory } from "../../types/SvVoucherRequest";
+import { selectedBeneficiaryCategorySelector } from "../../store/reducers/voucherRequest";
+import {
+  navigateToSvKoCheckIncomeThresholdScreen,
+  navigateToSvSickSelectDestinationScreen
+} from "../../navigation/actions";
+import I18n from "../../../../../i18n";
+import ButtonDefaultOpacity from "../../../../../components/ButtonDefaultOpacity";
+
+type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+const SickCheckIncomeScreen = (props: Props): React.ReactElement | null => {
```

Comment:
`SickCheckIncomeScreen` and `WorkerCheckIncomeScreen` will differ in something or will be the same? In the latter case could be an unique screen?

Reply:
The only difference is the path they should follow once the user press on the continue button.
I took the choice of 2 different screens because I would like to avoid every routing logic inside the screen.

Do you think it's however better use a single screen?

Reply:
Oh sorry, you're right, we also talked about this and the idea to use a common component wrapped by those two screens that will handle the routing!